assets/app/styles/_core.less

```diff
@@ -76,6 +66,12 @@ html, body {
     margin-top: 22px;
   }
 }
+#brand-logo {
+  background: url('@{image-path}/logo-origin-thin.svg');
```

Comment:
Use `background-image` instead of the shorthand style since we're only setting one property.